packages/flutter/lib/src/services/raw_keyboard_windows.dart

```diff
@@ -0,0 +1,291 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+import 'keyboard_key.dart';
+import 'keyboard_maps.dart';
+import 'raw_keyboard.dart';
+
+/// Platform-specific key event data for Windows.
+///
+/// This object contains information about key events obtained from Windows's
+/// `NSEvent` interface.
```

Comment:
Seems unlikely that it's from `NSEvent` ;)

Reply:
🤦‍♂